eCourse_backend/templates/registration/login.html

```diff
@@ -1,6 +1,120 @@
+{% extends "base.html" %}
+
+{% block additional-head %}
+<style>
+    input {
+        max-width: 400px;
+    }
+
+    .input-group-text {
+        width: 40px;
+    }
+    .errorIcon {
+        font-size: 2em;
+    }
+</style>
+{% endblock %}
+
+{% block navbar-content %}
 <h2>Login</h2>
+<ul class="navbar-nav">
+    <li class="nav-item">
+        <a class="nav-link" href="#">Sign up</a> <!-- TODO: make work -->
+    </li>
+</ul>
+{% endblock %}
+
+
+{% block content %}
+<!-- TODO: Rückmeldungen -->
+{% if form.errors %}
+<div class="container-fluid border-dark border" style="background-color: lightgray;">
+    <div class="row">
+        <div class="col-sm-1 align-content-center">
+            <span class="bi bi-exclamation-octagon-fill text-danger errorIcon"></span>
+        </div>
+        <div class="col-sm-11">
+            <h5>Username and Password didn't match</h5>
+            <h5>Please try again!</h5>
+        </div>
+    </div>
+</div>
+{% endif %}
+
+{% if next %}
+    {% if user.is_authenticated %}
+    <div class="container-fluid border-dark border" style="background-color: lightgray;">
+        <div class="row">
+            <div class="col-sm-1 align-content-center">
+                <span class="bi bi-exclamation-octagon-fill text-danger errorIcon"></span>
+            </div>
+            <div class="col-sm-11">
+                <h5>Your account doesn't have access to this page.</h5>
+                <h5>To proceed, please login with an account that has access.</h5>
+            </div>
+        </div>
+    </div>
+    {% else %}
+    <div class="container-fluid border-dark border" style="background-color: lightgray;">
+        <div class="row">
+            <div class="col-sm-1 align-content-center">
+                <span class="bi bi-exclamation-octagon-fill text-danger errorIcon"></span>
+            </div>
+            <div class="col-sm-11">
+                <h5>Please login to see this page.</h5>
+            </div>
+        </div>
+    </div>
+    {% endif %}
+{% endif %}
+
+<!-- Formular -->
 <form method="post">
     {% csrf_token %}
-    {{ form.as_p }}
-    <button type="submit">Do Stuff</button>
```

Comment:
* ```form.as_p``` doesn't work in this case, since this doesn't allow us to assign bs4 classes
* But a template was implemented (similar to the tutorial you listed), that takes a form and visualises it correctly (using the accessor methods for the fields of the form)

Reply:
Ok. Then lgtm